app/views/fsss_view.py

```diff
@@ -15,8 +20,13 @@ def get_emp_info(name, on_false={}, on_missing='Unspecified'):
 
  
     devices, device_numbers = process_device_details(device_information)
+#   raise Exception(str(devices))
```

Comment:
should these be commented out?

Reply:
Yes but I`ll delete them now (they were just so I could see  the value of the "devices" var